### ISSUE TITLE
HW5 having problems making my image move

### ISSUE THREAD

**Issue by mprimeau1** [Montana-Media-Arts/120_CreativeCoding#146]
So I made a pretty sweet image,  but I can't seem to get it to animate. I've tried to do several things but what I really wanted to start with was moving my  whole image with the cursor. I've made my cursor disapear  which is kind of cool.  Any suggestions of what I'm doing wrong? Heres a link of what I've done so far [`week5`](https://github.com/mprimeau1/120v2/tree/master/hw5)

**Comment by DVSnell** [comment]
Just taking a glance, I do see that the first few `push`es lack matching `pop`s.

**Comment by mprimeau1** [comment]
Ok, thanks, I looked at it and fixed those pops

**Comment by mprimeau1** [comment]
Any idea on the cursor thing?  This is the start of my sketch where I think I'm suppose to have the cursor translate in.

function setup(){

//background
    createCanvas( 600,600 );
    let angle = 0;
    background('blue');
    noCursor();

    function setup(){
      frameRate(20); //20 frames per second
    }
    function draw(){
      var fr = frameRate();
      print(fr);

    }
    
// Penguin with balloon

  push();

// make the penquin follow the mouse.
  translate('mouseX, mouseY');
  pop();

// penquin body

**Comment by wattse13** [comment]
@mprimeau1 You have two apostrophes surrounding your mouseX and mouseY parameters in your translate function. Those might be causing a problem. I also created a sketch that is supposed to follow the mouse using the translate function and when I added apostrophes, like the ones in your post above, my sketch stopped following the mouse. 
I would also try removing the `push();` and `pop();` sandbox that your `translate( mouseX, mouseY );` function is in.

**Comment by mprimeau1** [comment]
@wattse13 I did what you have said. Still no animation. Do you think I have the translate (mousex, mousey ); in the wrong place?

**Comment by wattse13** [comment]
@mprimeau1 Possibly? Would it be possible for you to upload screenshots of your code?

**Comment by mprimeau1** [comment]
Yeah, here's the start of it. I think that this is where the problem is somewhere.  
![help](https://user-images.githubusercontent.com/42810766/46032625-67ef0300-c0b9-11e8-9214-6cb44e7d89a9.png)

**Comment by wattse13** [comment]
It might be because you have background function within the setup function block. So instead of drawing the background again after every frame it is drawing the background once and only once at the beginning. I would move the `background();` and `noCursor();` functions to the `function draw(){}` block. At the moment, your `function draw();` block is also embedded in your `function setup(){}` block as there is a } missing around line 7. 
When I rewrote the code like the screenshot I've included, the balloon moved with the mouse.
![screenshot10](https://user-images.githubusercontent.com/42781660/46033774-3c214c80-c0bc-11e8-8b4e-078d60101195.png)
